install/recodex-core.service

```diff
@@ -0,0 +1,24 @@
+# ReCodEx core-api systemd configuration file
```

Comment:
The naming of the file is really very confusing, I would expect that unit with name `recodex-core` runs the REST Api ... and it does not, which is ... well, confusing

Reply:
I am open to suggestions. The service itself is closely attached to the core functionality, although the core is run by php-fpm which stands as a separate service. I thought that introducing another name would be even more confusing.